### PR TITLE
Update zh-CN translations

### PR DESCRIPTION
**Pull request by auroursa**
This PR is for improving the translation of Simplified Chinese and merging the latest translated strings.

### CODE REVIEW

**Comment by auroursa** [comment]
Please help me check this out @mitian233 @RitsukiP @U2FsdGVkX1

File: src/locale/locales/zh-CN/messages.po [hunk 1]
```diff
@@ -4236,7 +4198,7 @@ msgstr "显示回复"
 
 #: src/view/screens/PreferencesThreads.tsx:100
 msgid "Show replies by people you follow before all other replies."
-msgstr "在所有其他回复之前显示你关注的人的回复。"
+msgstr "在所有其他回复之前显示你关注的用户的回复。"
```
Comment (IceCodeNew) [inline src/locale/locales/zh-CN/messages.po:4201]:
```suggestion
msgstr "将你关注的用户的回复置于其他回复之前。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 2]
```diff
@@ -3969,7 +3931,7 @@ msgstr "选择以下一些账户进行关注"
 
 #: src/components/ReportDialog/SubmitView.tsx:133
 msgid "Select the moderation service(s) to report to"
-msgstr "你要将该条举报提交给哪位限制服务提供者？"
+msgstr "你要将该条举报提交给哪位限制服务提供者"
```
Comment (IceCodeNew) [inline src/locale/locales/zh-CN/messages.po:3934]:
```suggestion
msgstr "请选择你要向哪位限制服务提供者提交举报"
```

File: src/locale/locales/zh-CN/messages.po [hunk 3]
```diff
@@ -3568,7 +3536,7 @@ msgstr "删除转发"
 
 #: src/view/com/posts/FeedErrorMessage.tsx:202
 msgid "Remove this feed from your saved feeds"
-msgstr "将这个信息流从保存的信息流列表中删除？"
+msgstr "将这个信息流从保存的信息流列表中删除"
```
Comment (IceCodeNew) [inline src/locale/locales/zh-CN/messages.po:3539]:
```suggestion
msgstr "从保存的信息流列表中删除这个信息流"
```

File: src/locale/locales/zh-CN/messages.po [hunk 4]
```diff
@@ -2122,11 +2090,11 @@ msgstr "如果你根据你所在国家的法律定义还不是成年人，则你
 
 #: src/view/screens/ProfileList.tsx:612
 msgid "If you delete this list, you won't be able to recover it."
-msgstr "如果你删除此列表，将无法恢复"
+msgstr "如果你删除此列表，将无法恢复。"
```
Comment (IceCodeNew) [inline src/locale/locales/zh-CN/messages.po:2093]:
```suggestion
msgstr "该列表删除后将无法恢复。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 5]
```diff
@@ -2122,11 +2090,11 @@ msgstr "如果你根据你所在国家的法律定义还不是成年人，则你
...
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:342
 msgid "If you remove this post, you won't be able to recover it."
-msgstr "如果你移除此列表，将无法恢复"
+msgstr "如果你移除此列表，将无法恢复。"
```
Comment (IceCodeNew) [inline src/locale/locales/zh-CN/messages.po:2097]:
```suggestion
msgstr "该列表删除后将无法恢复。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 6]
```diff
@@ -1497,7 +1473,7 @@ msgstr "启用媒体播放器"
 
 #: src/view/screens/PreferencesFollowingFeed.tsx:147
 msgid "Enable this setting to only see replies between people you follow."
-msgstr "启用此设置以仅查看你关注的人之间的回复。"
+msgstr "启用此设置以仅查看你关注的用户之间的回复。"
```
Comment (IceCodeNew) [inline src/locale/locales/zh-CN/messages.po:1476]:
```suggestion
msgstr "启用此设置以便仅查看你关注的用户的回复。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 7]
```diff
@@ -464,7 +456,7 @@ msgstr "已屏蔽帖子。"
 
 #: src/screens/Profile/Sections/Labels.tsx:163
 msgid "Blocking does not prevent this labeler from placing labels on your account."
-msgstr "屏蔽不能阻止这个人在你的账户上放置标记"
+msgstr "屏蔽不能阻止这个用户在你的账户上放置标记。"
```
Comment (IceCodeNew) [inline src/locale/locales/zh-CN/messages.po:459]:
```suggestion
msgstr "屏蔽这个用户不能阻止他继续标记你的账户。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 8]
```diff
@@ -2227,7 +2195,7 @@ msgstr "邀请码：1 个可用"
 
 #: src/screens/Onboarding/StepFollowingFeed.tsx:65
 msgid "It shows posts from the people you follow as they happen."
-msgstr "他会显示你所关注的人发布的帖子。"
+msgstr "它将会显示你所关注的用户所发布的帖子。"
```
Comment (IceCodeNew) [inline src/locale/locales/zh-CN/messages.po:2198]:
```suggestion
msgstr "这将会显示你所关注的用户所发布的帖子。"
```

**Comment by IceCodeNew** [comment]
Thanks for inviting me for the review @auroursa, I would like to apply to the translate team and contribute to the translation.

File: src/locale/locales/zh-CN/messages.po [hunk 9]
```diff
@@ -672,7 +664,7 @@ msgstr "查看一些推荐的用户。关注他们还将推荐相似的用户。
 
 #: src/screens/Login/LoginForm.tsx:262
 msgid "Check your email for a login code and enter it here."
-msgstr ""
+msgstr "查看发送至你电子邮箱的验证码，并在此处输入。"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:667]:
```suggestion
msgstr "在这里输入刚才发送到你电子邮箱里的验证码"
```

File: src/locale/locales/zh-CN/messages.po [hunk 10]
```diff
@@ -1339,7 +1315,7 @@ msgstr "例如：艺术家、爱狗人士和狂热读者。"
 
 #: src/lib/moderation/useGlobalLabelStrings.ts:43
 msgid "E.g. artistic nudes."
-msgstr "例如：裸露艺术"
+msgstr "例如：裸露艺术。"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:1318]:
```suggestion
msgstr "例如：艺术性的裸露"
```

File: src/locale/locales/zh-CN/messages.po [hunk 11]
```diff
@@ -1968,7 +1936,7 @@ msgstr "用户识别符"
 
 #: src/view/screens/AccessibilitySettings.tsx:103
 msgid "Haptics"
-msgstr ""
+msgstr "触觉"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:1939]:
```suggestion
msgstr "触感"
```

File: src/locale/locales/zh-CN/messages.po [hunk 12]
```diff
@@ -1198,23 +1182,15 @@ msgstr "暗淡"
...
-msgstr ""
-
-#: src/view/screens/Settings/index.tsx:697
-#~ msgid "Disable haptics"
-#~ msgstr "禁用触觉反馈"
-
-#: src/view/screens/Settings/index.tsx:697
-#~ msgid "Disable vibrations"
-#~ msgstr "禁用振动"
+msgstr "关闭触觉反馈"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:1193]:
```suggestion
msgstr "关闭触感反馈"
```

File: src/locale/locales/zh-CN/messages.po [hunk 13]
```diff
@@ -3078,15 +3046,15 @@ msgstr "开启流程以登录到你现有的 Bluesky 账户"
...
+msgstr "开启 GIF 选择对话框"
 
 #: src/view/com/modals/InviteCodes.tsx:173
 msgid "Opens list of invite codes"
 msgstr "开启邀请码列表"
 
 #: src/view/screens/Settings/index.tsx:757
 msgid "Opens modal for account deletion confirmation. Requires email code"
-msgstr "开启用户删除确认界面，需要电子邮箱接收验证码。"
+msgstr "开启用户删除确认界面，需要电子邮箱接收验证码"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:3057]:
```suggestion
msgstr "需要邮件验证以继续进行账户删除操作"
```

File: src/locale/locales/zh-CN/messages.po [hunk 14]
```diff
@@ -4851,7 +4813,7 @@ msgstr "讨论串首选项"
 
 #: src/view/screens/Settings/DisableEmail2FADialog.tsx:102
 msgid "To disable the email 2FA method, please verify your access to the email address."
-msgstr ""
+msgstr "要关闭电子邮件两步验证，请先验证你的电子邮箱。"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:4816]:
```suggestion
msgstr "在关闭电子邮件两步验证前，请先验证你的电子邮箱地址。"
```

File: src/locale/locales/zh-CN/messages.po [hunk 15]
```diff
@@ -4872,7 +4834,7 @@ msgstr "切换以启用或禁用成人内容"
 #: src/screens/Hashtag.tsx:88
 #: src/view/screens/Search/Search.tsx:418
 msgid "Top"
-msgstr "最热门"
+msgstr "精选"
```
Comment (mitian233) [inline src/locale/locales/zh-CN/messages.po:4837]:
```suggestion
msgstr "热门"
```
精选一般用 Recommend

**Review by pfrazee — APPROVED** [review]
Thanks all!!

**Pull request merged**